interpretation/ww_head.py

```diff
@@ -677,19 +677,23 @@ def process(message, isGameMaster=False, isAdmin=False, isPeasant=False):
                 # TODO
                 return todo()
             if is_command(message, ['threaten', 'raven'], True) and user_role == "Raven":
-                # TODO
-                return todo()
+                msg = "**Usage:** Threaten a player.\n\n`" + prefix + "threaten <player>`\n\n"
+                msg += "**Example:** `" + prefix + "threaten @Randium#6521`\nThe command is compatible with emojis as a replacement for user mentions. "
+                msg += "This command can only be used by the Raven."
+                return [Mailbox().respond(msg,True)]
             if user_role == "Raven" and user_undead == 0:
                 help_msg += "`" + prefix + "threaten` - Threaten a player. (Raven only)\n"
 
             '''reveal'''
             # The Royal Knight's command
             if is_command(message, ['end', 'prevent', 'reveal', 'stop']) and user_role == "Royal Knight":
-                # TODO
+                # TODO             
                 return todo()
             if is_command(message, ['end', 'prevent', 'reveal', 'stop'], True) and user_role == "Royal Knight":
-                # TODO
-                return todo()
+                msg = "**Usage:** Prevent a lynch.\n\n`" + prefix + "prevent <player>`\n\n"
+                msg += "**Example:** `" + prefix + "prevent @Randium#6521`\nThe command is compatible with emojis as a replacement for user mentions. "
+                msg += "This command can only be used by the Royal Knight."
```

Comment:
LOOOOOOL you can't add a target to your lynch prevention. 😂 Either you prevent the lynch or you don't, there's no addition of a user!

Reply:
oh XD